### DISCUSSION TITLE
核显调用异常，可能无法硬解

### DISCUSSION THREAD

**Discussion by ShellShield** [ywmoyue/biliuwp-lite#534] [bug]
### Bug 描述

核显调用异常导致卡顿。
设备为surface go 初代，核显为英特尔hd 615，系统版本win11-23h2

### 操作步骤

```plain text
点击任意视频，点击开始播放或全屏时稳定发生掉帧（但此时声音不会卡顿），此时cpu占用会突然飙增，疑似是无法调用核显硬解导致cpu软解；并且在播放视频时会不定时出现轻微卡顿丢帧现象（此时同样声音不发生卡顿）。
上述两种情况无论切换何种画质何种解码方式均无法改善.
```

### 期望结果

应用能够稳定地播放视频且不发生卡顿掉帧现象

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

win11-23h2

### 设备类型

平板

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ShellShield** [comment]
当前软件版本4.6.27，这个问题从我最早用biliuwp就存在了，不太确定是不是我电脑的问题，我以前的surface pro5也有这个问题

**Comment by ywmoyue** [comment]
有空我用自己的surface go测一下

**Comment by ywmoyue** [comment]
<img width="900" alt="屏幕截图 2024-07-06 230323" src="https://github.com/ywmoyue/biliuwp-lite/assets/28388300/52c803a6-5bb6-40dd-a26f-5140c9b78f83">

我用我的surface go 初代测了没发现这个问题，看着是有正常硬解的，但我的系统是win10的，不清楚是不是win11导致的问题

**Reply by ywmoyue** [reply]
话说surface go 4g或8g的内存真能跑得动win11吗
